aries-site/src/pages/components/all-components.mdx

```diff
@@ -0,0 +1,586 @@
+import {
+  AccordionExample,
+  AnchorExample,
+  AvatarExample,
+  BoxExample,
+  ButtonStatesExample,
+  CalendarExample,
+  CarouselExample,
+  CheckBoxSimpleExample,
+  CheckBoxGroupSimpleExample,
+  ClockExample,
+  DataChartExample,
+  DateInputExample,
+  DiagramExample,
+  DropExample,
+  DropButtonExample,
+  FileInputSimpleExample,
+  FooterExample,
+  GridExample,
+  HeaderExample,
+  HeadingExample,
+  ImageExample,
+  LayerSideDrawerExample,
+  ListIconIdentifierExample,
+  MainExample,
+  MarkdownExample,
+  MaskedPhoneExample,
+  MenuExample,
+  MeterExample,
+  NavExample,
+  ParagraphExample,
+  PaginationExample,
+  RadioButtonGroupExample,
+  RangeInputExample,
+  RangeSelectorExample,
+  SelectExample,
+  SidebarExample,
+  SimpleCard,
+  SimpleSignUpExample,
+  SpinnerExample,
+  StackExample,
+  TableExample,
+  TabStatesExample,
+  TextSizeExample,
+  TextAreaExample,
+  TextInputExample,
+  ToolTipSimpleExample,
+  VideoExample,
+  WorldMapExample,
+} from '../../examples';
+import { Example } from '../../layouts';
+import { DocsMessage } from '../../components';
+
+On this page, you will find a comprehensive list of all Grommet components. The HPE Design System currently provides guidance and theming for a large subset of these components, but some components still do not have HPE Design System specific guidance. In those cases, you can still reference the Grommet docs for information on the components and their properties.
+
```

Comment:
Insert a "Component Index" or "Component Inventory" here.

Originally, I was thinking just links to each category, but found having each component listed was quite helpful, especially to see items presented side-by-side.

I started down a brute force markdown approach, but thinking we may want to execute as a presentation component which dynamically generates the name and links from `structure.js`. Each list item could contain component name, link, maybe an indicator signaling inclusion in the Design System or Grommet only?